package.json

```
@@ -55,6 +55,7 @@
   },
   "dependencies": {
     "patch-package": "^6.2.2",
-    "postinstall-postinstall": "^2.1.0"
+    "postinstall-postinstall": "^2.1.0",
+    "tallyho-onboarding": "^1.0.2"
```

not needed now that it's in `packages/hdwallet-tally/package.json`
```suggestion
    "postinstall-postinstall": "^2.1.0"
```